sensor_msgs/PointCloud2 cloud_out
float64[16] transform_mat
float32 x
float32 y
float32 z
float32 id
